spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -30,6 +35,11 @@ public class GravitinoDriverPlugin implements DriverPlugin {
   private static final Logger LOG = LoggerFactory.getLogger(GravitinoDriverPlugin.class);
 
   private GravitinoCatalogManager catalogManager;
+  private static final String[] GRAVITINO_DRIVER_EXTENSIONS =
+      new String[] {
+        GravitinoIcebergSparkSessionExtensions.class.getName(),
```

Comment:
Is there any restrict on the order of the extensions?

Reply:
Yes, `GravitinoIcebergSparkSessionExtensions` must be injected before `IcebergSparkSessionExtensions`.
Because we also need to rewrite some commands later, such as `SetWriteDistributionAndOrderingExec`, if `IcebergSparkSessionExtensions` is injected before `GravitinoIcebergSparkSessionExtensions`, spark will be an exception, because our table is `SparkIcebergTable` rather than `SparkTable`.

https://github.com/apache/iceberg/blob/01bc864b8eb8c4ca4240af85f00f4e67c78c466e/spark/v3.4/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/SetWriteDistributionAndOrderingExec.scala#L66

Reply:
ok.
i will grant the orders.

Reply:
> please add the comment about the restrict. and `registerSqlExtensions` seems does not grant the orders?

`registerSqlExtensions` have grant the orders using `LinkedHashSet`.